.github/workflows/deploy-canary.yml

```diff
@@ -0,0 +1,30 @@
+name: Canary Release
+
+on:
+  push:
```

Comment:
Does this really have to be on push? That means a release commit like this can also trigger it: https://github.com/firebase/firebase-js-sdk/commit/e50a4e906a135e12e582a53574abb4ab66c44521

I think you probably need something a bit more advanced like:

```
on:
  pull_request:
    branches:
      - master
    types:
      - closed

jobs:
  deploy:
    if: github.event.pull_request.merged
```

Reply:
Was copying the Travis config but this seems better, changed.